### DISCUSSION TITLE
Issue with Yum repos when creating Searchnode

### DISCUSSION THREAD

**Discussion by Edelweiss1996** [Security-Onion-Solutions/securityonion#8713] [Unsupported Versions] [answered]
Over Saturday and Sunday I have tried to set up a SO Distributed Network within AWS. The manager created with no issues, it goes through the 'Pre-Flight' checks and installs correctly. The manager appears to have no faults, is accessible within the console and begins to generate logs related to it's components such as OSSEC and Kratos. I don't believe the manager has any faults.

However I tried to then set up a Search Node as is required. I have done through through the .160 AMI that is available and also a standard version of Centos7 with the repository pulled from git. In both cases these machines fail the installation process.

When installation begins it starts basic OS Updates after the 'Pre-flight' using the standard Yum Repositories and does it's updates fine. However as we continue the installation process it enables the Security Onion Repositories **(which are the same as on the manager)** and begins to fail with timeout errors: `https://repo.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 12] Timeout on https://repo.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: (28, 'Connection timed out after 30001 milliseconds')`

We compared the repository information to the manager's using yum-config-manager and we do not see any difference. The manager is able to call yum updates and it will search through the repositories fine. AWS security groups are identical to avoid any issues for now, but are fairly locked down but the manager built successfully with these rules in place.

I am not sure where to exactly start with this: The manager built correctly from what appear to be the same repositories, they have the same firewall rules. They both have a internet connection and can connect. I can force through a reinstall using so-setup-network however it fails the preflight checks due to the Yum hangup.

**Comment by dougburks** [comment, marked as answer]
I did several installations yesterday and didn't have any problems pulling from repo.securityonion.net. Double-check your AWS security groups and any other firewall config.

**Reply by Edelweiss1996** [reply]
Hi there, I think it's a case of problem in chair, not in computer. Had someone else go through the installation process with the same settings and setup. Did the exact same as me and it built fine. Not sure what precisely caused this. Same SG and firewall settings.
